config/environments/test.rb

```diff
@@ -41,6 +41,12 @@
   # config.active_record.mass_assignment_sanitizer = :strict
 
   config.action_mailer.default_url_options = { :host => "localhost", :port => 3000 }
+
+  config.middleware.use ExceptionNotification::Rack,
+    email: {
+      sender_address: "exception@test.com",
+      exception_recipients: ["fake_developer@test.com"]
+    }
```

Comment:
It's good. Just use one of the safe testing domains like `example.com` (ref: https://www.rfc-editor.org/rfc/rfc2606.txt)